src/main/java/com/moddy/server/service/model/ModelService.java

```diff
@@ -0,0 +1,112 @@
+package com.moddy.server.service.model;
+
+
+import com.moddy.server.common.exception.enums.ErrorCode;
+import com.moddy.server.common.exception.model.NotFoundException;
+import com.moddy.server.controller.model.dto.response.ModelMainResponse;
+import com.moddy.server.controller.model.dto.response.OfferResponse;
+import com.moddy.server.domain.designer.Designer;
+import com.moddy.server.domain.hair_model_application.repository.HairModelApplicationJpaRepository;
+import com.moddy.server.domain.hair_service_offer.HairServiceOffer;
+import com.moddy.server.domain.hair_service_offer.repository.HairServiceOfferJpaRepository;
+import com.moddy.server.domain.model.ModelApplyStatus;
+import com.moddy.server.domain.model.repository.ModelJpaRepository;
+import com.moddy.server.domain.prefer_offer_condition.OfferCondition;
+import com.moddy.server.domain.prefer_offer_condition.PreferOfferCondition;
+import com.moddy.server.domain.prefer_offer_condition.repository.PreferOfferConditionJpaRepository;
+import com.moddy.server.domain.user.User;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Sort;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+
+@Service
+@RequiredArgsConstructor
+@Transactional(readOnly = true)
+public class ModelService {
+
+    private final ModelJpaRepository modelJpaRepository;
+    private final HairModelApplicationJpaRepository hairModelApplicationJpaRepository;
+    private final HairServiceOfferJpaRepository hairServiceOfferJpaRepository;
+    private final PreferOfferConditionJpaRepository preferOfferConditionJpaRepository;
+
+    private Page<HairServiceOffer> findOffers(Long userId, int page, int size){
+        PageRequest pageRequest = PageRequest.of(page-1, size, Sort.by(Sort.Direction.DESC,"id"));
+        Page<HairServiceOffer> offerPage = hairServiceOfferJpaRepository.findByUserId(userId, pageRequest);
+
+        return offerPage;
+    }
+
+    private ModelApplyStatus calModelStatus(boolean apply, boolean offer){
+
+        //테스트 코드에 false true 경우 있는지 테스트하기
+        if(apply == false && offer == false) {
+            return ModelApplyStatus.NOTHING;
+        }
+        else if (apply == true && offer == false) {
+            return ModelApplyStatus.APPLY;
+        }
+        else if (apply == true && offer == true) {
+            return ModelApplyStatus.APPLY_AND_OFFER;
+        }
```

Comment:
```suggestion
        if(!apply && !offer) return ModelApplyStatus.NOTHING;
        else if (apply && !offer) return ModelApplyStatus.APPLY;
        else if (apply && offer) return ModelApplyStatus.APPLY_AND_OFFER;
```
p2
이런 식으로 코드를 단축할 수 있을 것 같습니다

Reply:
오 이렇게 쓸 생각을 못해봤네요 습관이 무서운법.... 감사합니다~~! 더 간단하고 좋은거 같아요